Formula/g/gnome-latex.rb

```diff
@@ -32,23 +32,39 @@ class GnomeLatex < Formula
   depends_on "gettext" => :build
   depends_on "gobject-introspection" => :build
   depends_on "itstool" => :build
-  depends_on "pkg-config" => :build
+  depends_on "pkg-config" => [:build, :test]
```

Comment:
Is this accurate?

Reply:
not accurate, let me remove the test part